### PR TITLE
Development

### PR DESCRIPTION
**Pull request by odebretseni**

### CODE REVIEW

File: tasks/install-docker.yml [hunk 1]
```diff
@@ -77,3 +66,14 @@
   become_user: "{{ work_user }}"
   command: open --background -a Docker
   changed_when: false
+
+- name: Change docker-compose settings
```
Comment (BrunIF) [inline tasks/install-docker.yml]: This is docker settings.
Try replace to `"credsStore" : "desktop"`

**Review by BrunIF — CHANGES_REQUESTED** [review]

File: tasks/install-docker.yml [hunk 2]
```diff
@@ -77,3 +66,14 @@
   become_user: "{{ work_user }}"
```
Comment (BrunIF) [inline tasks/install-docker.yml]: `become_user` without `become: yes` havn't sense.

**Review by BrunIF — CHANGES_REQUESTED** [review]

File: tasks/install-docker.yml (hunk 2 above)
Comment (odebretseni) [reply tasks/install-docker.yml]: there are become directives - "become: true"

File: tasks/install-yarn.yml [hunk 3]
```diff
@@ -1,6 +1,6 @@
 ---
 - name: Copy yarn install script
-  template: src=yarn.sh dest=/Users/{{ work_user }} mode=0777
+  template: src=yarn.sh.j2 dest=/Users/{{ work_user }}/yarn.sh mode=0777
```
Comment (BrunIF) [inline tasks/install-yarn.yml]: install with npm

File: tasks/install-nvm.yml [hunk 4]
```diff
@@ -18,3 +18,9 @@
   become: false
   command: /bin/zsh -c "source ~/.zshrc"
   changed_when: false
+
+- name: Run install protractor
+  shell: . ~/.nvm/nvm.sh && which npm && npm install -g protractor
+  environment:
+    PATH: "/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin"
```
Comment (BrunIF) [inline tasks/install-nvm.yml]: {{ lookup('env','PATH') }}

**Review by BrunIF — CHANGES_REQUESTED** [review]

File: tasks/install-docker.yml (hunk 1 above)
Comment (odebretseni) [reply tasks/install-docker.yml]: This setting is by default and just doesn't work with it, therefore I delete it

**Review by BrunIF — APPROVED** [review]

**Pull request merged**